src/promptflow-core/promptflow/_utils/flow_utils.py

```diff
@@ -75,15 +75,25 @@ def resolve_flow_path(
     else:
         flow_path = Path(flow_path)
 
+    if flow_path.is_dir():
+        target_folder = flow_path
+        target_filer = FLOW_FLEX_YAML if (target_folder / FLOW_FLEX_YAML).is_file() else FLOW_DAG_YAML
+        if (target_folder / FLOW_DAG_YAML).is_file() and (target_folder / FLOW_FLEX_YAML).is_file():
```

Comment:
let's refactor to only do two is_file call here

Reply:
done